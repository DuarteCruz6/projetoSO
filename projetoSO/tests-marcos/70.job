SHOW
WRITE [(i6,94)(o5,41)(v2,51)(a4,76)(g0,1)]
WRITE [(f4,72)(m6,62)(t6,19)(f9,55)]
READ [f9]
WRITE [(g9,91)(z0,16)]
READ [m6]
BACKUP
WRITE [(q7,13)(a1,58)(c2,30)]
WRITE [(z8,39)(h8,3)]
WRITE [(r4,75)(q5,69)(b4,11)(n1,51)]
WRITE [(i4,83)]
WRITE [(a5,62)(r8,54)(x8,62)(r5,49)(g7,80)]
WRITE [(x2,89)]
WRITE [(q1,6)(o1,65)]
WRITE [(g4,26)(f6,99)(h2,66)(h9,20)(n6,46)]
READ [i4]
WRITE [(n2,39)(g8,40)]
WRITE [(o0,16)(y6,21)(s2,24)(n9,45)]
WRITE [(u1,17)(r2,47)]
BACKUP
READ [v2]
WRITE [(n5,22)]
READ [i4]
SHOW
WRITE [(v3,20)(f0,76)(n0,4)(t4,86)(p0,78)]
WRITE [(j7,47)(i0,20)(q4,26)(d7,52)]
DELETE [r4,g4,a4,r8]
WRITE [(j0,70)(w7,8)]
WRITE [(i3,48)(n8,74)]
DELETE [x2,z0,g9,i0,v2]
READ [h2]
WAIT 3000
DELETE [n0,x8,s2,i6]
WRITE [(c5,83)]
WRITE [(k8,91)]
DELETE [y6,c5,h2]
DELETE [w7,o1,n2]
WRITE [(y1,65)(b3,75)(a8,14)(a3,100)]
READ [q7]
WAIT 3000
WRITE [(d8,89)(i5,2)]
WRITE [(c4,62)(p5,44)(o2,92)]
WRITE [(x1,65)(g3,64)(u9,27)(h6,100)(m4,91)]
WRITE [(t1,36)(q0,97)(o6,49)(f8,14)]
READ [t1]
DELETE [f9]
READ [n8]
BACKUP
WRITE [(q9,77)(e1,5)(b1,85)(r1,73)(r3,24)]
READ [n5]
WRITE [(l9,46)(u3,93)(x5,39)]
WRITE [(e0,97)(e8,84)(l1,25)(b8,88)(g4,71)]
WRITE [(p6,81)]
WRITE [(z2,18)]
WRITE [(o9,56)(d5,45)(d0,8)(u8,86)]
WRITE [(j1,80)(y4,4)]
WRITE [(d6,8)(o3,97)(e6,16)(p1,35)]
WRITE [(z9,5)(i9,18)(y7,44)]
