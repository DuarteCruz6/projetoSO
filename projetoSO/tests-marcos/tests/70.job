SHOW
WRITE [(m3,46)(l6,27)]
SHOW
WRITE [(c4,93)]
READ [m3]
DELETE [l6]
DELETE [c4]
WRITE [(h3,23)(l8,19)(q9,7)(k8,81)]
WRITE [(s8,15)(p5,72)(r9,3)(p8,68)]
DELETE [q9,p8,k8,s8]
WRITE [(t4,46)(s0,85)(m5,96)(h5,93)(w4,51)]
WRITE [(a7,92)(f9,49)(r1,17)(l4,67)(o9,60)]
WRITE [(d9,81)(d7,34)(r5,58)(f7,12)]
BACKUP
READ [d9]
SHOW
WRITE [(t9,16)(k1,9)(n2,70)(s9,51)(o3,67)]
SHOW
WRITE [(e6,27)(t6,89)(t1,31)(l6,91)(k8,54)]
DELETE [h3,n2,s0,t6,r5]
WRITE [(e8,59)]
READ [l6]
READ [l4]
WRITE [(b0,38)]
WRITE [(l5,0)]
WRITE [(k0,93)(f4,14)(e0,51)(f1,70)(y8,42)]
READ [t4]
BACKUP
WRITE [(g4,99)(x7,33)(q2,79)(g0,57)(c0,37)]
DELETE [m5]
WRITE [(o0,13)(c8,29)(g7,79)(m7,87)]
DELETE [k8,e8,w4,o3]
READ [c8]
BACKUP
WRITE [(p0,33)(y3,49)(s0,82)(m1,21)(q0,55)]
WRITE [(j6,29)(k2,73)]
WRITE [(z5,28)]
READ [m3]
BACKUP
READ [e0]
DELETE [r9,r1,f9]
READ [k1]
WRITE [(d1,91)]
WRITE [(o1,46)(v7,80)(g5,15)(u9,31)(e4,57)]
DELETE [z5,x7,u9]
BACKUP
WRITE [(i3,69)(x5,79)(s6,26)]
WRITE [(r0,90)]
WRITE [(u8,54)(x7,28)]
DELETE [f7]
WRITE [(w5,75)(z8,75)(z2,5)(j7,65)]
WRITE [(y2,19)]
WRITE [(k6,94)(j4,17)(c3,14)(w7,82)(l3,62)]
READ [c0]
READ [c3]
WRITE [(i5,15)(q7,26)]
WRITE [(a0,35)(i7,7)(t0,48)(a4,74)(g3,21)]
DELETE [g4,c3,o9]
WRITE [(x9,47)(p3,7)]
SHOW
SHOW
READ [b0]
DELETE [m7]
WRITE [(n9,76)(d0,63)(u0,63)(f7,25)]
SHOW
DELETE [s6,t1,r0,u0,i7]
WRITE [(r3,63)]
WRITE [(n5,5)(q4,6)]
WRITE [(a6,48)(r5,65)(v5,59)(i4,25)]
READ [n9]
WRITE [(j9,32)(r2,4)(f2,35)(r2,63)]
READ [y2]
WRITE [(l0,54)(n8,40)(p2,1)(g8,23)(y6,56)]
WRITE [(h7,11)(l1,58)(m2,6)]
WRITE [(t3,24)(w8,12)]
WRITE [(f5,50)(s2,67)(g1,71)]
WAIT 1000
WRITE [(i2,38)(f3,81)(c7,47)(f8,0)(x3,55)]
WAIT 1000
READ [k2]
READ [t9]
WRITE [(b5,35)(u2,94)]
WRITE [(x6,77)(v8,93)(t2,41)]
WRITE [(e9,3)(w6,27)(m6,29)(p1,2)(j3,11)]
WRITE [(e7,54)]
DELETE [a6,f4,t3,q2]
WRITE [(n2,80)(c5,16)(r6,92)]
WRITE [(i1,84)(c2,37)(e3,50)(j0,22)(v2,97)]
WRITE [(b6,60)(k8,38)(h6,95)(f9,73)]
WRITE [(k7,39)(d4,21)]
WRITE [(w0,91)(l9,72)(n7,75)]
WAIT 1000
BACKUP
READ [e0]
WRITE [(s1,15)]
WRITE [(m4,16)(x0,71)(q1,77)]
WRITE [(o8,77)(a8,75)(o4,3)(n6,26)]
