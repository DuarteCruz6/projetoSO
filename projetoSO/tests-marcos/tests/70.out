(l6, 27)
(m3, 46)
[(m3,46)]
[(d9,81)]
(a7, 92)
(d7, 34)
(d9, 81)
(f7, 12)
(f9, 49)
(h5, 93)
(h3, 23)
(l4, 67)
(l8, 19)
(m5, 96)
(m3, 46)
(o9, 60)
(p5, 72)
(r5, 58)
(r1, 17)
(r9, 3)
(s0, 85)
(t4, 46)
(w4, 51)
(a7, 92)
(d7, 34)
(d9, 81)
(f7, 12)
(f9, 49)
(h5, 93)
(h3, 23)
(k1, 9)
(l4, 67)
(l8, 19)
(m5, 96)
(m3, 46)
(n2, 70)
(o3, 67)
(o9, 60)
(p5, 72)
(r5, 58)
(r1, 17)
(r9, 3)
(s9, 51)
(s0, 85)
(t9, 16)
(t4, 46)
(w4, 51)
[(l6,91)]
[(l4,67)]
[(t4,46)]
[(c8,29)]
[(m3,46)]
[(e0,51)]
[(k1,9)]
[(c0,37)]
[(c3,14)]
(a4, 74)
(a0, 35)
(a7, 92)
(b0, 38)
(c8, 29)
(c0, 37)
(d1, 91)
(d7, 34)
(d9, 81)
(e4, 57)
(e0, 51)
(e6, 27)
(f1, 70)
(f4, 14)
(g3, 21)
(g5, 15)
(g7, 79)
(g0, 57)
(h5, 93)
(i7, 7)
(i5, 15)
(i3, 69)
(j4, 17)
(j7, 65)
(j6, 29)
(k6, 94)
(k2, 73)
(k0, 93)
(k1, 9)
(l3, 62)
(l5, 0)
(l6, 91)
(l4, 67)
(l8, 19)
(m1, 21)
(m7, 87)
(m3, 46)
(o1, 46)
(o0, 13)
(p3, 7)
(p0, 33)
(p5, 72)
(q7, 26)
(q0, 55)
(q2, 79)
(r0, 90)
(s6, 26)
(s0, 82)
(s9, 51)
(t0, 48)
(t1, 31)
(t9, 16)
(t4, 46)
(u8, 54)
(v7, 80)
(w7, 82)
(w5, 75)
(x9, 47)
(x7, 28)
(x5, 79)
(y2, 19)
(y3, 49)
(y8, 42)
(z2, 5)
(z8, 75)
(a4, 74)
(a0, 35)
(a7, 92)
(b0, 38)
(c8, 29)
(c0, 37)
(d1, 91)
(d7, 34)
(d9, 81)
(e4, 57)
(e0, 51)
(e6, 27)
(f1, 70)
(f4, 14)
(g3, 21)
(g5, 15)
(g7, 79)
(g0, 57)
(h5, 93)
(i7, 7)
(i5, 15)
(i3, 69)
(j4, 17)
(j7, 65)
(j6, 29)
(k6, 94)
(k2, 73)
(k0, 93)
(k1, 9)
(l3, 62)
(l5, 0)
(l6, 91)
(l4, 67)
(l8, 19)
(m1, 21)
(m7, 87)
(m3, 46)
(o1, 46)
(o0, 13)
(p3, 7)
(p0, 33)
(p5, 72)
(q7, 26)
(q0, 55)
(q2, 79)
(r0, 90)
(s6, 26)
(s0, 82)
(s9, 51)
(t0, 48)
(t1, 31)
(t9, 16)
(t4, 46)
(u8, 54)
(v7, 80)
(w7, 82)
(w5, 75)
(x9, 47)
(x7, 28)
(x5, 79)
(y2, 19)
(y3, 49)
(y8, 42)
(z2, 5)
(z8, 75)
[(b0,38)]
(a4, 74)
(a0, 35)
(a7, 92)
(b0, 38)
(c8, 29)
(c0, 37)
(d0, 63)
(d1, 91)
(d7, 34)
(d9, 81)
(e4, 57)
(e0, 51)
(e6, 27)
(f7, 25)
(f1, 70)
(f4, 14)
(g3, 21)
(g5, 15)
(g7, 79)
(g0, 57)
(h5, 93)
(i7, 7)
(i5, 15)
(i3, 69)
(j4, 17)
(j7, 65)
(j6, 29)
(k6, 94)
(k2, 73)
(k0, 93)
(k1, 9)
(l3, 62)
(l5, 0)
(l6, 91)
(l4, 67)
(l8, 19)
(m1, 21)
(m3, 46)
(n9, 76)
(o1, 46)
(o0, 13)
(p3, 7)
(p0, 33)
(p5, 72)
(q7, 26)
(q0, 55)
(q2, 79)
(r0, 90)
(s6, 26)
(s0, 82)
(s9, 51)
(t0, 48)
(t1, 31)
(t9, 16)
(t4, 46)
(u0, 63)
(u8, 54)
(v7, 80)
(w7, 82)
(w5, 75)
(x9, 47)
(x7, 28)
(x5, 79)
(y2, 19)
(y3, 49)
(y8, 42)
(z2, 5)
(z8, 75)
[(n9,76)]
[(y2,19)]
[(k2,73)]
[(t9,16)]
[(e0,51)]
